### PR TITLE
Update install-policy-framework.md

### PR DESCRIPTION
**Pull request by dockerymick**
changes for the open source website

### CODE REVIEW

File: content/getting-started/install-policy-controllers.md [hunk 1]
```diff
@@ -13,98 +13,115 @@ After policy framework is installed, you could install the policy controllers to
...
+   iam-policy-controller-7c5f746866-v65jb   1/1     Running   0          2m43s
+   ...
+   ```
+   
+The IAM policy controller is installed.
 
 ## What is next
 
-After a successful deployment, test the policy framework and configuration policy controller with a sample policy. Run the following command:
+1. After a successful deployment, test the policy framework and configuration policy controller with a sample policy. Run the following command:
```
Comment (ycao56) [inline content/getting-started/install-policy-controllers.md:124]: can you add some spacing for the shell scripts below as others? it is misaligned
image

File: content/getting-started/install-policy-framework.md [hunk 2]
```diff
@@ -13,61 +13,67 @@ After hub and klusterlet is installed, you could install the policy framework co
...
+
+Complete the following steps to install the policy framework from prebuild images on Quay.io:
+
+1. Clone the `governance-policy-framework` repository:
+
+   ```Shell
+   git clone https://github.com/open-cluster-management/governance-policy-framework.git
+   ```
+
+2. Deploy the policy framework components to the hub cluster with the following commands:
```
Comment (ycao56) [inline content/getting-started/install-policy-framework.md:40]: 2 and 4 are misaligned
image
can you fix it?

**Comment by dockerymick** [comment]
@ycao56 Thanks for the feedback, i made a couple of adjustments let me know if it is aligned appropriately

**Comment by ycao56** [comment]
looks good!

**Review by ycao56 — APPROVED** [review]

**Comment by ycao56** [comment]
/assign @qiujian16
/assign @mikeshng 
please review and merge.

**Comment by qiujian16** [comment]
commit needs signoff

**Comment by mikeshng** [comment]
@dockerymick could you please rebase and sign off on your commits/PR. Ping me on slack if you need some help regarding the dco sign off part. Thanks.

**Comment by dockerymick** [comment]
Closing this PR so that I can sign-off as requested. Thanks @mikeshng for helping with the sign-off! I appreciate you

**Pull request closed without merging**